code/game/machinery/stasis.dm

```diff
@@ -146,7 +145,7 @@
 	if(stasis_running())
 		if(!IS_IN_STASIS(L_occupant))
 			chill_out(L_occupant)
-	else if(IS_IN_STASIS(L_occupant))
+	else if(IS_IN_STASIS(L_occupant) || L_occupant.health < HEALTH_THRESHOLD_FULLCRIT)
```

Comment:
I still don't see the point of this nerf to stasis beds. You've already made them space-ruin/admin-only, why nerf them further?